RELEASE-NOTES.md

```diff
@@ -1,5 +1,27 @@
 # Prefect Release Notes
 
+## Release 2.10.17
+
+### Enhancements
+- Update PrefectClient to follow redirects by default — https://github.com/PrefectHQ/prefect/pull/9988
+- Add `CustomWebhookNotificationBlock` for general purpose notification — https://github.com/PrefectHQ/prefect/pull/9547
+
+### Fixes
+- Limit the number of files concurrently opened by `prefect deploy` when searching for flows — https://github.com/PrefectHQ/prefect/pull/10014
```

Comment:
```suggestion
- Limit the number of files concurrently opened by `prefect deploy` when searching for flows — https://github.com/PrefectHQ/prefect/pull/10014
- Fix js error when running prefect in an unsecure context (http vs https) — https://github.com/PrefectHQ/prefect-ui-library/pull/1501
```

Reply:
@pleek91 Can you rephrase 
"Fix js error when running prefect in an unsecure context (http vs https)" as what that means for the user?

Reply:
What page does it cause to break in the Prefect UI?

Reply:
Just the ui in general, its not specific to a single page